test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -47,7 +47,7 @@ var (
 
 	tests = []struct{ nodes, existingPods, minPods int }{
 		{nodes: 500, existingPods: 500, minPods: 1000},
-		{nodes: 5000, existingPods: 5000, minPods: 1000},
+		{nodes: 5000, existingPods: 15000, minPods: 1000},
```

Comment:
I am not sure if we should do this. People only do profiling when they need. At that point they probably will customize the tests anyway to get the results they want.

I am leaning towards making the benchmark easy and fast to run for most of the cases. In rare cases when a developer needs to analyze the performance in more details such as profiling, they may need to customize the tests.

Reply:
The difference is not that significant in terms of e2e runtime, few additional minutes I think is ok. Also, people who deliberately run those benchmarks are by definition trying to analyze performance, and profiling is a core tool for doing that.

Reply:
hmm, I was mainly running the affinity benchmarks, and those were not impacted much by increasing the number of existingpods, but the storage based benchmarks are stuck. I will revert this change for now, hopefully the new benchmark suite will give us better way to profile and compare performance across test cases.

Reply:
SG. We can revisit this later. Even if 15k takes too long, we can choose to skip it in presubmit